code/modules/admin/verbs/ping_all_admins.dm

```diff
@@ -0,0 +1,39 @@
+/client/proc/ping_all_admins()
+	set name = "Ping all admins"
+	set category = "Admin"
+
+	if(!check_rights(R_ADMIN, FALSE))
+		return
+
+	var/msg = input(src, "What message do you want the ping to show?", "Ping all admins") as text|null
+	msg = sanitize(copytext(msg, 1, MAX_MESSAGE_LEN))
+
+	if(!msg)
+		return
+
+	var/list/admins_to_ping = list()
+
+	for(var/client/C in GLOB.admins)
+		if(R_ADMIN & C.holder.rights)
```

Comment:
should this be the other way around? Or does it work both ways?

Reply:
It works both ways